Mathlib/GroupTheory/GroupAction/Hom.lean

```diff
@@ -177,12 +220,13 @@ theorem ofEq_apply {φ' : M → N} (h : φ = φ') (f : X →ₑ[φ] Y) (a : X) :
 variable {ψ χ} (M N)
 
 /-- The identity map as an equivariant map. -/
+@[to_additive "The identity map as an equivariant map."]
 protected def id : X →[M] X :=
   ⟨id, fun _ _ => rfl⟩
 
 variable {M N Z}
 
-@[simp]
+@[to_additive, simp]
```

Comment:
```suggestion
@[to_additive (attr := simp)]
```
same below

Reply:
I had wondered whether I should do this, which the linter doesn't impose, but would have been logical in view of other tags. But then, why should I do it?

Reply:
Because if you don't do this, then only the multiplicative versions are `simp`.

Reply:
oops! thank you!